packages/pagination/src/NextPage.tsx

```diff
@@ -0,0 +1,86 @@
+import { Button } from '../../button/src/index.js';
+import { classNames } from "@chbphone55/classnames";
+import { i18n } from '@lingui/core';
+import { pagination as ccPagination } from '@warp-ds/css/component-classes';
+import { usePagination } from './PaginationContainer.js';
+import IconChevronRight16 from '@warp-ds/icons/react/chevron-right-16';
+import React from 'react';
+
+type NextPageProps = {
+  /**
+   * @default Next page
+   */
+  'aria-label'?: string;
+
+  /**
+   * Butto label to render on mobile
+   */
+  children?: React.ReactNode;
+
+  /** Additional CSS class for the element. */
+  className?: string;
+
+  /** The href for the anchor element. */
+  href: string;
+
+  /** Additional CSS styles for the element. */
+  style?: React.CSSProperties;
+
+  /** onClick function */
+  onClick: (event: React.UIEvent<HTMLElement>) => void;
+};
+
+const NextPage = React.forwardRef<
+  React.AnchorHTMLAttributes<HTMLAnchorElement>,
+  NextPageProps
+>(({ children, className, ...props }, ref) => {
+  const { currentPage, lastPage } = usePagination();
+
+  if (currentPage >= lastPage) {
+    return null;
+  }
+
+  const ariaLabel = i18n._({
+    id: 'pagination.aria.next-page',
+    message: 'Next page',
+    comment:
+      'Default screenreader message for next page button in the pagination component',
+  });
+
+  const label =
+    children ??
+    i18n._({
+      id: 'pagination.button.next-page',
+      message: 'Next page',
+      comment:
+        'Default message for next page button in the pagination component',
+    });
+
+  return (
+    <>
+      {currentPage <= 1 && (
+        <Button
+          link
+          {...props}
+          ref={ref}
+          rel="next nofollow"
+          className={classNames(ccPagination.firstPageButton, className)}
+        >
+          <span className={ccPagination.firstPageLabel}>{label}</span>
+        </Button>
+      )}
+      <Button
+        pill
+        aria-label={ariaLabel}
+        {...props}
+        ref={ref}
+        rel="next nofollow"
+        className={classNames(ccPagination.nextPage, className)}
```

Comment:
Maybe that's intended, but I saw that on mobile when I press Next/PrevPage button, the hover background color remains visible until I click elsewhere.
![pagination-mobile](https://github.com/warp-ds/react/assets/41303231/0d48401e-d40c-4c7c-b50d-6c97be9b2ebb)
<img width="304" alt="Screenshot 2024-01-08 at 13 23 13" src="https://github.com/warp-ds/react/assets/41303231/871bcfb8-5479-48aa-a308-3963fb49742f">

Reply:
It doesn't matter since the page would have changed/reloaded. You can see this on the result page on tori/finn